deep_qa/data/token_indexers/token_indexer.py

```diff
@@ -75,7 +74,6 @@ def pad_token_sequence(self,
         """
         raise NotImplementedError
 
-    @staticmethod
-    def from_params(params: Params):
-        choice = params.pop_choice('type', list(token_indexers.keys()), default_to_first_choice=True)
-        return token_indexers[choice].from_params(params)
```

Comment:
Where are you putting this code? This was intentional, actually, and makes constructing objects from params a lot easier. If it's not here, it has to be duplicated every time you want to construct an object.